fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/serialization/JsonParserHelper.java

```diff
@@ -469,7 +470,8 @@ public static LocalDateTime convertDateTimeFrom(final String dateTimeAsString, f
         if (StringUtils.isNotBlank(dateTimeAsString)) {
             try {
                 DateTimeFormatter formatter = new DateTimeFormatterBuilder().parseCaseInsensitive().appendPattern(dateTimeFormat)
-                        .toFormatter(clientApplicationLocale);
+                        .parseDefaulting(ChronoField.HOUR_OF_DAY, 0).parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
+                        .parseDefaulting(ChronoField.SECOND_OF_MINUTE, 0).toFormatter(clientApplicationLocale);
                 eventLocalDateTime = LocalDateTime.parse(dateTimeAsString, formatter);
```

Comment:
java.time.format.DateTimeParseException is returned when LocalDateTime.parse()  is called on a Date string lacking a time portion like '23 July 2020' (dont know why convertDateTimeFrom is called instead of  convertFrom for such values). Hence added parseDefaulting for time part when it is missing. 
Will check for other such occurrences.